lib/rucio/api/rse.py

```diff
@@ -57,7 +60,7 @@ def add_rse(rse, issuer, deterministic=True, volatile=False, city=None, region_c
     return rse_module.add_rse(rse, deterministic=deterministic, volatile=volatile, city=city,
                               region_code=region_code, country_name=country_name, staging_area=staging_area,
                               continent=continent, time_zone=time_zone, ISP=ISP, rse_type=rse_type, latitude=latitude,
-                              longitude=longitude, ASN=ASN, availability=availability)
+                              longitude=longitude, ASN=ASN, availability=availability, supported_checksums=supported_checksums)
```

Comment:
I would not add this in any of the `add_rse` at all. It should just stay a pure optional rse attribute. The `del_rse_checksum`, `add_rse_checksum` etc. are also not necessary, it can all be done via the `add_rse_attribute`. That would be more in line with other "configuration" attributes we have.

Reply:
In my opinion `del_rse_checksum ` and `add_rse_checksum` are much needed to use the correct key syntax.
I agree on this peculiar call, this is a left around...